packages/core/core/src/requests/DevDepRequest.js

```diff
@@ -163,11 +165,22 @@ export function invalidateDevDeps(
   }
 }
 
-export async function runDevDepRequest(
-  api: RunAPI,
+type DevDepRequestResult = {|
+  specifier: DependencySpecifier,
+  resolveFrom: ProjectPath,
+  hash: string,
+  additionalInvalidations: void | Array<{|
+    range?: ?SemverRange,
+    resolveFrom: ProjectPath,
+    specifier: DependencySpecifier,
+  |}>,
+|};
+
+export async function runDevDepRequest<TResult>(
+  api: RunAPI<TResult>,
   devDepRequest: DevDepRequest,
 ) {
-  await api.runRequest<null, void>({
+  await api.runRequest<null, DevDepRequestResult | void>({
     id: 'dev_dep_request:' + devDepRequest.specifier + ':' + devDepRequest.hash,
     type: 'dev_dep_request',
     run: ({api}) => {
```

Comment:
Also strange: this function has a `api.storeResult` call but AFAICT that value is actually never used/read anywhere

Reply:
does removing it break something?

Reply:
Nevermind, the result is loaded here
https://github.com/parcel-bundler/parcel/blob/ea61ad9dc295133eeb50cce3e1665f4dfc08b632/packages/core/core/src/requests/AssetRequest.js#L80-L90